src/Components/Backoffice/Users/BackofficeUsers.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import {Table, Thead, Tbody, Tr, Th, Td} from 'react-super-responsive-table'
+import 'react-super-responsive-table/dist/SuperResponsiveTableStyle.css'
+import "../../../Styles/BackOfficerUsersTable.css";
+import Button from "@material-ui/core/Button/Button";
+import { Link } from 'react-router-dom';
+
+const BackofficeUsers = () => {
+    const createData =(nombre, id, email,) => {
```

Comment:
maybe this functionallity should be on users service, createUserObject or something like that so it can be reused by other views